src/components/parameter.jsx

```diff
@@ -30,6 +30,12 @@ class Parameter extends React.Component {
             ? <p className="description">{parameter.description}</p>
             : <em><FormattedMessage id="noDescription"/></em>
           }
+          { parameter.documentation &&
+            (<span><FormattedMessage id="documentationText"/></span>)
+          }
+          { parameter.documentation &&
```

Comment:
This block should be merged in the previous one, as the condition is the same.
You may need to wrap the `span` and `p` in a higher-level HTML element, as React don't like juxtaposing elements, but that should be doable 🙂

Reply:
Oops. Done.